frontend/src/composables/useToaster.ts

```diff
@@ -0,0 +1,13 @@
+import { useToast } from 'primevue/usetoast';
+
+export async function useToaster(callback: Function, options: any = {}) {
+  const toast = useToast();
+  try {
+    await callback();
+  } catch (error: any) {
+    const { severity = 'error', summary = 'Unable to load data.', detail = error, life = 5000 } = options;
+    toast.add({ severity: severity, summary: summary, detail: detail, life: life });
+  }
+
+  return;
```

Comment:
Just noting for future use we could take in an array of callbacks and await each, and also expose a `loading` from this composable so the parent using it would know when the function(s) have completed. Can think about putting that stuff in later.